### ISSUE TITLE
"Find all references" functionality

### ISSUE THREAD

**Issue by retailcoder** [retailcoder/Rubberduck#337]
Once #336 is done, implementing a "Find all references" context menu to the *Code Explorer* will be a breeze.

We also need to add this functionality to the code pane's context menu, which would allow the user to right-click any declaration and bring up a *search results* window that allows navigating to all references of the selected identifier.

**Comment by retailcoder** [comment]
Duplicate of #333
